### PR TITLE
acceptance: ignore container removal errors in CircleCI

### PR DESCRIPTION
**Pull request by tamird**
[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4297)

### CODE REVIEW

**Comment by tschottdorf** [comment]
LGTM

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4297#-:-KAC0_8hdOSdQtbkJo8J:676834233)*

File: acceptance/cluster/docker.go [hunk 1]
```diff
@@ -125,16 +118,18 @@ func maybePanic(err error) {
...
 // container.
 func (c *Container) Remove() error {
+	err := c.cluster.client.ContainerRemove(types.ContainerRemoveOptions{
+		ContainerID:   c.id,
+		RemoveVolumes: true,
+	})
+
 	if os.Getenv("CIRCLECI") == "true" {
-		// HACK: Removal of docker containers fails on circleci with the error:
+		// HACK: Removal of docker containers on circleci reports the error:
```
Comment (petermattis) [inline acceptance/cluster/docker.go]: Has the `docker` on circleci upgraded? The hack might no longer be necessary.

Comment (tamird) [inline acceptance/cluster/docker.go]: It definitely still spews an error.

Comment (petermattis) [inline acceptance/cluster/docker.go]: Oh well, the hack stays.

**Pull request merged**